crates/miden-lib/asm/kernels/transaction/lib/account.masm

```diff
@@ -614,7 +614,7 @@ end
 #! - no map is found for ROOT.
 export.set_map_item.12
     # store index for later
-    dup loc_store.0
+    movup.2 loc_store.0
```

Comment:
This change would move a field element of the `KEY` to the top of the stack, which is not what we want here. The original code should still be correct, duplicating the index at the top of the stack.